src/config/default-app-config.ts

```diff
@@ -385,4 +386,15 @@ export class DefaultAppConfig implements AppConfig {
     enabled: false,
     mathjax: false,
   };
+
+  // Which vocabularies should be used for which search filters
+  // and whether to show the filter in the search sidebar
+  // Take a look at the filter-vocabulary-config.ts file for documentation on how the options are obtained
+  vocabularies: FilterVocabularyConfig[] = [
+    {
+      filter: 'subject',
+      vocabulary: 'srsc',
+      enabled: true
```

Comment:
Tiny note.  I think we should default this to `false`.  Otherwise, it's a bit difficult to disable, as you need to provide a full configuration like this:
```
vocabularies:
  - filter: 'subject'
    vocabulary: 'srsc'
    enabled: false
```

There doesn't seem to be an easier way to disable it by simply setting `vocabularies: false` or `vocabularies: [ ]`.  (Both those options cause errors)

So, because it's not easy to turn off, I think it should be turned off by default.  We can document how to enable this easily for `srsc` for sites that need it.